includes/multi-currency/MultiCurrency.php

```diff
@@ -609,6 +610,7 @@ public function get_selected_currency(): Currency {
 		$code = $this->get_stored_currency_code();
 
 		$code = $this->compatibility->override_selected_currency() ? $this->compatibility->override_selected_currency() : $code;
+		$code = apply_filters( 'wcpay_multi_currency_get_selected_currency', $code );
```

Comment:
not sure but probably existing unit test should be modified to cover this case

Reply:
Fixed.